README.md

```diff
@@ -54,3 +55,7 @@ The adapter repositories remain in `/home/vagrant/`.
 ### This does not seem to work on my machine
 
 Even though most hardware supports virtualization, your CPU may not or you may need to enable it in your BIOS/UEFI settings.
+
+### There is no GUI
+
+In case you killed the session before provisioning finished, the setup of your VM might be incomplete. You might still be able to interact with the VM without the GUI. However, the easiest way to fix this is to run `vagrant destroy` and start from the beginning.
```

Comment:
```suggestion
In case you killed the session before provisioning finished, the setup of your VM might be incomplete. You might still be able to interact with the VM without the GUI. In that case, run `vagrant up --provision`.
```